src/core/mesh/qgsmesheditor.h

```diff
@@ -76,6 +79,13 @@ class CORE_EXPORT QgsMeshEditor : public QObject
     //! Returns TRUE if a \a face can be added to the mesh
     bool faceCanBeAdded( const QgsMeshFace &face );
 
+    /**
+     * Returns TRUE if the face is geometrically compatbile, that is not intersect or contins any other elements (faces or vertices).
```

Comment:
compatbile - typo
contins - typo

`Returns TRUE if the face does not intersect or contains any other elements (faces or vertices)`